services/web-app/pages/api/log-request.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { Logging } from '@carbon-platform/api/logging'
+
+import { CARBON_INTERNAL_API_SECRET } from '@/config/constants'
+
+const logging = new Logging('web-app', 'log-request-api')
+
+const logRequest = (req, res) => {
+  if (!req.body) {
+    res.status(400).send('Bad request (no body)')
+    return
+  }
+
+  const { logMessage, internalApiSecret } = req.body
+
+  if (!logMessage) {
+    res.status(400).send('Bad request (no logMessage)')
+    return
+  }
+
+  if (!internalApiSecret || internalApiSecret !== CARBON_INTERNAL_API_SECRET) {
+    logging.warn(`unauthorized access from ${req.socket.remoteAddress}`)
```

Comment:
```suggestion
```

Let's just silently ignore these so external users can't flood the logs by submitting malformed requests.